Resources/Prototypes/Catalog/Fills/Crates/permaescape.yml

```diff
@@ -0,0 +1,396 @@
+- type: entity
+  name: Perma Escape Crate Spawner
+  id: CratePermaEscapeSpawner
+  parent: CrateEmptySpawner
+  components:
+  - type: RandomSpawner
+    prototypes:
+      - CratePermaEscapeDigging
+      - CratePermaEscapeDigging
+      - CratePermaEscapeEVA
+      - CratePermaEscapeGun
+      - CrateEngineeringMiniJetpack
+      - CratePermaEscapeBureaucracy
+      - CratePermaEscapeLights
+      - CrateTrashCartFilled
+      - CratePermaEscapeMats
+      - CratePermaEscapeMats
+      - CratePermaEscapeGiftsFromSyndicate
+      - CratePermaEscapeMerc
+      - CratePermaEscapeComs
+      - CratePermaEscapeComs
+      - CratePermaEscapeTowercap
+      - CratePermaEscapeTowercap
+      - CrateServiceCustomSmokable
+      - ClosetMaintenanceFilledRandom
+      - ClosetMaintenanceFilledRandom
+      - ClosetMaintenanceFilledRandom
+    rarePrototypes:
+      - MobTick # These need to be killable by one dude with a shovel.
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobTick
+      - MobCarp
+      - MobCarp
+      - MobCarp
+      - MobCarpMagic
+      - MobKangarooSpace
+      - MobBearSpace
+    rareChance: .30
```

Comment:
Plz no just refactor it to have counts if you need but this is awful to maintain.

Reply:
Trade offer: Only MobTick x1 but nothing else (I was on the fence about the others anyway since I think they'll just murder the aforementioned dude with a shovel).